script.js

```diff
@@ -145,10 +145,17 @@ recipesApp.errorHandle = function () {
 
 }
 
+recipesApp.cardListener = function() {
+    const cardClick = document.querySelector('#cardHolder')
+    cardClick.addEventListener('click', function (event) {
+        const innerCard = document.querySelector('#cardHolderInner').classList.toggle('rotateCard');
```

Comment:
Really nice event listener and idea for flipping the recipe card! looking forward to more styling! :)